frontend/src/components/SamplesList.tsx

```diff
@@ -131,6 +131,13 @@ export default function SamplesList({
     const fieldName = params.colDef.field!;
     const { oldValue, newValue, node: rowNode } = params;
 
+    function resetAlertIfCostCentersAreAllValid(changes: SampleChange[]) {
+      const allRowsHaveValidCostCenter = changes.every(
+        (c) => c.fieldName !== "costCenter" || isValidCostCenter(c.newValue)
+      );
+      if (allRowsHaveValidCostCenter) setAlertContent(defaultAlertContent);
+    }
```

Comment:
We're using the alert message state as an indicator for whether to disable the "Submit Updates" button. I defined this function here and am calling it in places where we can enable it again below.